staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -576,6 +603,38 @@ func (g *Cloud) ensureInternalInstanceGroups(name string, nodes []*v1.Node) ([]s
 	return igLinks, nil
 }
 
+func gceInstanceNames(instances []*gceInstance) []string {
+	ret := make([]string, len(instances))
+	for i, instance := range instances {
+		ret[i] = instance.Name
+	}
+	return ret
+}
+
+// reuseInstanceGroup returns true if all the instances in the ig are part of the instancesToInclude and list of instances in the ig.
```

Comment:
Fix comment "reuseInstanceGroupForInternalLoadbalancer returns".  Also include a comment about the other 2 return values.

Reply:
done 3b484c45d3d